lua/nvim-treesitter/parsers.lua

```diff
@@ -617,6 +617,16 @@ list.fsh = {
   maintainers = { "@mgramigna" },
 }
 
+list.fsharp = {
+  install_info = {
+    url = "https://github.com/ionide/tree-sitter-fsharp",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
```

Comment:
```suggestion
```
Not needed if that's the default branch (but you do need to add a lockfile.json entry).